### ISSUE TITLE
defaultSelections Prop is not working correctly

### ISSUE THREAD

**Issue by flashnt** [adammcarth/react-native-segmented-picker#23]
Here is my code:
```javascript
import React, { Component } from 'react';
import SegmentedPicker from 'react-native-segmented-picker';
import {Text, View} from 'react-native';
 
export default class Demo extends Component {
  constructor(props) {
    super(props);
    this.segmentedPicker = React.createRef();
  }
 
  onConfirm = (selections) => {
    console.info(selections);
  }
 
  render() {
    return (
      <View>
        <Text
            style={{marginTop: 300}}
            onPress={() => this.segmentedPicker.current.show()}>
            Click me
        </Text>

        <SegmentedPicker
            ref={this.segmentedPicker}
            onConfirm={this.onConfirm}
            defaultSelections={{'col_1': 'option_3', 'col_2': 'option_5'}}
            options={[
            {
                key: 'col_1',
                items: [
                    { label: 'Option 1', value: 'option_1' },
                    { label: 'Option 2', value: 'option_2' },
                    { label: 'Option 3', value: 'option_3' },
                ],
            },
            {
                key: 'col_2',
                items: [
                    { label: 'Option 4', value: 'option_4' },
                    { label: 'Option 5', value: 'option_5' },
                ],
            },
            ]}
        />
      </View>
    );
  }
}
```
Here is the unexpected behavior:
![problems](https://user-images.githubusercontent.com/55541791/86771785-6a033580-c085-11ea-93cc-8baf2ebdee9c.gif)

When the screen is first focused, the default selections are not reflected in the picker. However, after I confirm the current choice and open the picker again, the default selections are reflected in the picker.

**Comment by adammcarth** [comment]
Hey @flashnt. Thanks for the report.

How very strange! The code you pasted above appears to be totally correct... and there's a whole bunch of passing E2E tests which assert that the default selection functionality is working on Android.

I'm thinking this might actually be a race condition/performance problem that needs to be optimised for.

Can I ask what simulator and Android version you're running this on?

**Comment by flashnt** [comment]
@adammcarth I used Genymotion Samsung Galaxy S10 Android 9.0 to test the above code

**Comment by flashnt** [comment]
Hi @adammcarth , do you have any updates on the issue?

**Comment by adammcarth** [comment]
Hi @flashnt,

Apologies for not getting back to you sooner - it's been a busy couple of weekends here.

I have finally been able to replicate the bug using your code. I think I understand where the issue is happening and will have an update about a fix in the next couple of days.

Sorry for the inconvenience!

**Comment by adammcarth** [comment]
@flashnt This issue should be resolved in `2.0.1`.

Please let me know if you're still experiencing problems after updating the package.

**Comment by flashnt** [comment]
Hi @adammcarth, it is working as expected for me now. Thanks for your reply.
